### PR TITLE
track in workflow button

### PR DESCRIPTION
**Pull request by akash1810**
# Screenshots
## When not tracked yet
image

## When tracked in Workflow:
image

TODO
- [x] screenshots
- [x] deploy to CODE
- [x] fix tests
- [x] detect production office

### CODE REVIEW

**Comment by akash1810** [comment]
[track in Workflow button](https://trello.com/c/0qyN0s3B/56-track-in-workflow-button)

File: public/video-ui/src/reducers/workflowReducer.js [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+      });
+    case 'WORKFLOW_STATUS_GET_RECEIVE':
+      return Object.assign({}, state, {
+        status: action.status || {}
+      });
+    case 'WORKFLOW_STATUS_NOT_FOUND':
+      return Object.assign({}, state, {
+        status: action.status
+      });
+    case 'WORKFLOW_GET_PRODUCTION_OFFICE':
```
Comment (Reettaphant) [inline public/video-ui/src/reducers/workflowReducer.js]: Does the production office need to be saved in the redux store? Could we just save it in the state of the workflow component and have the logic for computing it in a utility function?

File: public/video-ui/src/services/WorkflowApi.js [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+      url: `${WorkflowApi.workflowUrl}/api/sections`,
+      crossOrigin: true,
+      withCredentials: true
+    }).then(response => {
+      return response.data
+        .map(section => Object.assign({}, section, { title: section.name }))
+        .sort((a, b) => {
+          if (a.title.toLowerCase() < b.title.toLowerCase()) return -1;
+          if (a.title.toLowerCase() > b.title.toLowerCase()) return 1;
+          return 0;
```
Comment (Reettaphant) [inline public/video-ui/src/services/WorkflowApi.js]: Could we use `sortBy` from lodash instead here?

File: public/video-ui/src/reducers/workflowReducer.js (hunk 1 above)
Comment (akash1810) [inline public/video-ui/src/reducers/workflowReducer.js]: good point, doesn't really make sense to have it in the store

File: public/video-ui/src/services/WorkflowApi.js (hunk 2 above)
Comment (Reettaphant) [inline public/video-ui/src/services/WorkflowApi.js]: I guess it's just a bit neater but doesn't really make much of a difference.

Comment (akash1810) [inline public/video-ui/src/services/WorkflowApi.js]: The [lodash syntax](https://lodash.com/docs/4.17.4#sortBy) looks largely similar to this, I'm tempted to stay with the native `.sort` function. What do you think?

**Pull request merged**